app/components/Company/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
+import Rating from 'material-ui-rating';
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: '30px',
+    fontSize: '20px',
+  },
+  field: {
+    float: 'left',
+    width: '50%',
+  },
+});
+
+const Company = props => {
+  const { company } = props;
+  const classes = useStyles();
+
+  const renderCompany = () => {
+    if (!company) {
+      return null;
+    }
+
+    return (
+      <>
+        <div>
```

Comment:
You can create a component for this div. As you can see, you repeat this code 5 times. This is a good sign that it is time for a new component.